testsuite/test_shell.py

```diff
@@ -7,6 +7,26 @@
 from testsuite.support import ROOT_DIR, PseudoFile
 
 
+def safe_line_split(line):
+    """ parses the path, from message details """
+
+    # On certain platforms, the drive notation conflicts
```

Comment:
Also, these comments can / should probably just move into the function docstring.